indico/modules/events/contributions/templates/display/_contribution_list.html

```diff
@@ -0,0 +1,60 @@
+{% macro render_displayed_entries_fragment(displayed_num, total_num) -%}
+    <span id="filtering-state" class="i-button label icon-list"
+          title="{% trans %}Displaying {{displayed_num}} out of {{total_num}} contributions{% endtrans %}">
+        <strong>{{ displayed_num }}</strong> / {{ total_num }}
+    </span>
+{%- endmacro %}
+
+{% macro render_contribution_list(event, timezone, contributions, with_session_info=true) %}
+    {% for contrib in contributions -%}
+        <div class="contribution-row" data-friendly-id="{{ contrib.friendly_id }}">
+            <div class="contrib-title info" data-searchable="{{ contrib.title }}">
+                <span class="value">
+                    <a href="{{ url_for('contributions.display_contribution', contrib) }}">{{ contrib.title }}</a>
+                </span>
+            </div>
+            {% if contrib.description -%}
+                <div class="info">
+                    <span class="label">{% trans %}Description{% endtrans %}</span>
+                    <span class="value">{{ contrib.description|strip_tags }}</span>
```

Comment:
i don't think `strip_tags` is correct here. `description` returns an object (RichMarkup) that takes care of showing it properly, rendering some HTML as we still support it in descriptions but using pre-style linebreaks if needed